### DISCUSSION TITLE
What is the role of enableIndexedDBProvider in the preset.config.mjs file

### DISCUSSION THREAD

**Discussion by chenmins** [toeverything/AFFiNE#1764] [Q&A]
### What happened?

​preset.config.mjs enableIndexedDBProvider 的作用是什么？
如果 我希望用户登录后，默认创建的工作空间自动云同步，那么enableIndexedDBProvider 设置为false是否可以不启用indexdb的本地功能？如果不能，请问在哪里修改

### Version

app.affine.pro

### What browsers are you seeing the problem on?

Chrome

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by himself65** [comment]
`enableIndexedDBProvider` will save the workspace into the local browser but also fetch from the server when it responds. If you didn't set it to `true,` every time you fresh the page will fetch the workspace data from the server.
